packages-web/datagrid/src/ui/DatagridPreview.scss

```diff
@@ -0,0 +1,3 @@
+svg {
```

Comment:
feels too global, maybe at least scope it to datagrid

Reply:
fixed